man/tpm2_policyauthorize.1.md

```diff
@@ -90,7 +90,7 @@ verification of the signature on the pcr policy digest using **tpm2_policyauthor
 
 ## Create a TPM object like a sealing object with the authorized policy based authentication:
 * tpm2_createprimary -Q -a o -g sha256 -G rsa -o prim.ctx
-* tpm2_create -Q -g sha256 -u sealing_key.pub -r sealing_key.pub -I- -C prim.ctx\
+* tpm2_create -Q -g sha256 --u sealing_pubkey.pub -r sealing_prikey.pub -I- -C prim.ctx\
```

Comment:
You have a `--u` now, just `-u`

Reply:
I think -u is a correct option to get an output file which contains the public portion of the created object, as per man page.